scargo/commands/check.py

```diff
@@ -343,7 +343,7 @@ class CppcheckChecker(CheckerFixer):
     check_name = "cppcheck"
 
     def check_files(self) -> int:
-        cmd = "cppcheck --enable=all --suppress=missingIncludeSystem src/ main/"
+        cmd = "cppcheck --enable=all --suppress=missingIncludeSystem --inline-suppr src/ main/"
```

Comment:
src/ main/" - can we use instead variable defining source dir for the project?

Reply:
I guess we could, but it's entirely tangential to this PR.